### ISSUE TITLE
Fix Pill layout in Order Details view in mobile & tablet

### ISSUE THREAD

**Issue by D4ynx** [D4ynx/reids-bakery-hq#19]
Clean up pill layout for mobile and tablet view in Order Details. See photo for basis

> Production Status & Paid Status for order details view should be besides Order Number
> Usable buttons should be in one row, at the left side of the screen
> Position must be fixed when dynamically resized

<img width="801" height="258" alt="Image" src="https://github.com/user-attachments/assets/ebcd2c82-b49e-42a1-aba4-1900a6f9d56d" />
